Cynosura.Studio.CliTool/Commands/GenerateCommand.cs

```diff
@@ -0,0 +1,127 @@
+                Console.WriteLine($"Entity {entity.Name} generated successfully");
+            }
+            var enums = await accessor.GetEnumsAsync();
+            foreach (var en in enums)
+            {
+                await GenerateEnumAsync(en.Name);
+                Console.WriteLine($"Enum {en.Name} generated successfully");
+            }
+
+            return true;
+        }
+
+        private async Task<bool> GenerateEnumActionAsync(IEnumerable<string> args)
+        {
+            var ar = args as string[] ?? args.ToArray();
+            if (ar.Length != 1)
+            {
+                Console.WriteLine($"Command syntax: {CliApp.CommandName} generate enum <enumName>");
+                return false;
+            }
+            var name = ar.FirstOrDefault();
+            await GenerateEnumAsync(name);
+            Console.WriteLine($"Enum {name} generated successfully");
+            return true;
+        }
+
+        private async Task GenerateEnumAsync(string name)
+        {
+            var accessor = new SolutionAccessor(SolutionDirectory);
+            var enums = await accessor.GetEnumsAsync();
+            var generator = LifetimeScope.Resolve<CodeGenerator>();
+            var en = enums.FirstOrDefault(f => f.Name == name);
+            if (en == null)
+            {
+                throw new Exception($"Enum {name} not found");
+            }
+            await generator.GenerateEnumAsync(accessor, en);
+            await generator.GenerateEnumViewAsync(accessor, new View(), en);
+        }
+
+        private async Task<bool> GenerateEntityActionAsync(IEnumerable<string> args)
+        {
+            var ar = args as string[] ?? args.ToArray();
+            if (ar.Length != 1)
+            {
+                Console.WriteLine($"Command syntax: {CliApp.CommandName} generate entity <entityName>");
+                return false;
+            }
+            var name = ar.FirstOrDefault();
+            await GenerateEntityAsync(name);
+            Console.WriteLine($"Entity {name} generated successfully");
+            return true;
+        }
+
+        private async Task GenerateEntityAsync(string name)
+        {
+            var accessor = new SolutionAccessor(SolutionDirectory);
+            var entities = await accessor.GetEntitiesAsync();
+            var generator = LifetimeScope.Resolve<CodeGenerator>();
+            var entity = entities.FirstOrDefault(f => f.Name == name);
+            if (entity == null)
+            {
+                throw new Exception($"Entity {name} not found");
+            }
+            await generator.GenerateEntityAsync(accessor, entity);
+            await generator.GenerateViewAsync(accessor, new View(), entity);
+        }
+
+        public override string Help()
+        {
+            return $"{CliApp.CommandName} generate <action>\r\n" +
+                   $"Available actions: \r\n{string.Join("\r\n\t", _actions.Keys)}\r\n";
```

Comment:
Please align help messages accordingly. Especially when showing lists, sometimes first element is on the same line, and sometimes it's on the next line, but is not indented like other elements.

Reply:
Solved